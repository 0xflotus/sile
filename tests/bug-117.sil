\begin[papersize=a5]{document}
%nice script from SILE book
\begin{script}
for i=1,10 do
SILE.typesetter:typeset(i .. " x " .. i .. " = " .. i*i .. ". ")
SILE.typesetter:leaveHmode()
SILE.call("smallskip")
end
\end{script}
\end{document}
