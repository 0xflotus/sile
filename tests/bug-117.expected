<tests/bug-117.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.97638
My 	35.84288
Set font 	Gentium;10;200;regular;normal;;LTR
T	17	(1)
Mx 	48.34669
T	88	(x)
Mx 	56.05392
T	17	(1)
Mx 	63.42423
T	29	(=)
Mx 	70.49181
T	17	(1)
Mx 	75.18420
T	14	(.)
Mx 	40.97638
My 	50.84288
T	18	(2)
Mx 	48.34669
T	88	(x)
Mx 	56.05392
T	18	(2)
Mx 	63.42423
T	29	(=)
Mx 	70.49181
T	20	(4)
Mx 	75.18420
T	14	(.)
Mx 	40.97638
My 	65.84288
T	19	(3)
Mx 	48.34669
T	88	(x)
Mx 	56.05392
T	19	(3)
Mx 	63.42423
T	29	(=)
Mx 	70.49181
T	25	(9)
Mx 	75.18420
T	14	(.)
Mx 	40.97638
My 	80.84288
T	20	(4)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	20	(4)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	17 22	(16)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	95.84288
T	21	(5)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	21	(5)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	18 21	(25)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	110.84288
T	22	(6)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	22	(6)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	19 22	(36)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	125.84288
T	23	(7)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	23	(7)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	20 25	(49)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	140.84288
T	24	(8)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	24	(8)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	22 20	(64)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	155.84288
T	25	(9)
Mx 	48.34618
T	88	(x)
Mx 	56.05289
T	25	(9)
Mx 	63.42268
T	29	(=)
Mx 	70.48975
T	24 17	(81)
Mx 	79.87451
T	14	(.)
Mx 	40.97638
My 	170.84288
T	17 16	(10)
Mx 	53.03701
T	88	(x)
Mx 	60.74217
T	17 16	(10)
Mx 	72.80280
T	29	(=)
Mx 	79.86832
T	17 16 16	(100)
Mx 	93.94546
T	14	(.)
[1] Mx 	207.41759
My 	553.73265
T	17	(1)
End page
Finish

